### PR TITLE
Work with DirectMail Tokens for Newsletter Unsubscribe, if FEManager …

### PR DESCRIPTION
**Pull request by Xarno**
…is used for Newsletter Subscriptions.

Unsubscribelink in the Directmail template needs to contain these two tokens in this order: ###USER_uid######SYS_AUTHCODE###
And the need to be a get parameter named „key“. Resulting in: https://<mydomain>/<mypage>/unsubscribe/?key=12345

### CODE REVIEW

File: ext_emconf.php [hunk 1]
```diff
@@ -28,6 +28,7 @@
         'depends' => [
             'typo3' => '8.7.0-8.7.99',
             'php' => '7.0.0-7.2.99',
+            'direct_mail' => '5.2.2-5.9.99'
```
Comment (tmotyl) [inline ext_emconf.php]: In my POV direct_mail should not be a hard dependency of fe_manager.
If you need to make sure, that fe_manager is loaded later, after directmail (in Packagestates) it's enough to put direct mail into suggests array.

Comment (Xarno) [reply ext_emconf.php]: Yes, i would like it also optional. I have grabbed it from our fe manager extensions, so we have no code to have it optional in there.

Comment (chrcan) [reply ext_emconf.php]: Hello work this workoround also with direct_mail | 6.0.0-dev and femanager | 5.2.0

Comment (chrcan) [reply ext_emconf.php]: i mean this direct_mail https://github.com/jokumer/direct_mail/tree/TYPO3_9

**Comment by sbusemann** [comment]
I won't merge a direct dependency of femanager to direct_mail, as most User will not use direct_Mail.

**Comment by sbusemann** [comment]
no feedback received, so I close the branch.

**Pull request closed without merging**